src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.search {
+  position: relative;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: inherit;
+  box-sizing: border-box;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```

Comment:
Use f variable for colors you meet several times

Reply:
fix it 
<img width="415" alt="image" src="https://user-images.githubusercontent.com/99331793/203923084-b5987973-265f-4f6e-a335-c733d6c1dc00.png">